src/components/EmailItem/Index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+
+function EmailItem({ name, subject, body, date, read }) {
+  const [dateObj, setDateObj] = useState(new Date(date).toLocaleDateString());
+
+  return (
+    <div
+      className={`${
+        read === "1" ? "bg-slate-50 text-black" : "bg-gray-500 text-white"
+      } p-2.5 border-b border-r border-gray-200 hover:bg-blue-500`}
+    >
```

Comment:
```suggestion
  const readClassNames = (read === "1") ? "bg-slate-50 text-black" : "bg-gray-500 text-white"

  return (
    <div
      className={`${readClassNames} p-2.5 border-b border-r border-gray-200 hover:bg-blue-500`}
    >
```

OPTIONAL: Pulling these out into their own variable would make it easier to read - especially if you have lots of tailwind classes to add.